packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -197,8 +198,9 @@ export class SlackBot {
                     }
                 }
             } catch (e) {
-                if (this.lightdashConfig.mode === LightdashMode.PR)
-                    notifySlackError(e, l.url, client, event);
+                if (this.lightdashConfig.mode === LightdashMode.PR) {
+                    void notifySlackError(e, l.url, client, event);
```

Comment:
I am not sure if we should await for this or not. let me know and I can change!

Reply:
@rephus can you confirm if it's fine to omit await here?

Reply:
I don't think it matters, it seems this only happens in PR, you can leave it as it is